packages/devtools_app/lib/src/inspector/flutter/inspector_screen.dart

```diff
@@ -104,15 +104,28 @@ class _InspectorScreenBodyState extends State<InspectorScreenBody>
       decoration: BoxDecoration(
         border: Border.all(color: Theme.of(context).focusColor),
       ),
-      child: InspectorTree(
-        controller: summaryTreeController,
-        isSummaryTree: true,
+      child: GestureDetector(
+        child: InspectorTree(
+          controller: summaryTreeController,
+          isSummaryTree: true,
+        ),
+        onTapUp: (TapUpDetails tapUpDetails) {
+          summaryTreeController.setFocus();
```

Comment:
we should try hard to integrate with Flutter concepts of focus rather than adding our own concept of setting focus. Focus is something that is really hard to do robustly. Notice you will already get some flutter focus changes when you tab between controls in the flutter version of the inspector. the focus changes aren't good but that illustrates the baked in functionality.